inc/class-rewrites.php

```diff
@@ -42,7 +42,7 @@ public function hooks() {
 	 * @return string Potentially modified permalink
 	 */
 	public function strip_cloned_from_url( $permalink, $post ) {
-		if ( is_post_cloned( $post->ID ) ) {
+		if ( is_post_cloned( $post->ID ) && strip_cloned( $permalink, $post ) ) {
 			$permalink = preg_replace( '/(' . $this->search_string . ')\/$/', '', trailingslashit( $permalink ), 1 );
```

Comment:
```suggestion
			$permalink = preg_replace( '/(' . $this->search_string . ')\/$/', '/', trailingslashit( $permalink ), 1 );
```
I noticed it stripped the end slash from the permalink too, can fix that here as it's a related bug.